taichi/backends/opengl/opengl_data_types.h

```diff
@@ -57,13 +57,5 @@ inline int opengl_argument_address_shifter(DataType type) {
   return 3 - opengl_data_address_shifter(type);
 }
 
-inline int opengl_get_snode_meta_size(const SNode &snode) {
```

Comment:
I believe removing this could also save us one SSBO (However, I believe it is not used in mobile)